helm/kubeblocks/falkordb-cluster-omnistrate/tests/e2e_omnistrate/omnistrate_client/instance.py

```
@@ -0,0 +1,570 @@
+        self._fleet_api.handle_response(
+            response, f"Failed to delete instance {self.instance_id}"
+        )
+
+        if not wait_for_delete:
+            return
+
+        try:
+            self.wait_for_instance_status(
+                timeout_seconds=cfg["stop_timeout"]
+            )
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)